packages/dd-trace/src/remote_config/index.js

```diff
@@ -0,0 +1,260 @@
+'use strict'
+
+const uuid = require('crypto-randomuuid')
+const { EventEmitter } = require('events')
+const Scheduler = require('./scheduler')
+const tracerVersion = require('../../../../package.json').version
+const request = require('../exporters/common/request')
+const log = require('../log')
+
+const clientId = uuid()
+
+const POLL_INTERVAL = 5e3
+
+// There MUST NOT exist separate instances of RC clients in a tracer making separate ClientGetConfigsRequest
```

Comment:
Why is that the case? Should we ensure that this cannot be done instead of documenting to not do it?

Reply:
Because the agent and backend expect to only have 1 state per tracer, and doing otherwise would cause some unexpected message delivery inconsistencies I'd say.
Since tracer.init() already has double call protection, I'd say we don't really need to enforce it. The only reason this comment is here is because I know I have bad memory.